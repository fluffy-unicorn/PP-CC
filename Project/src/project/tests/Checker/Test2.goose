# Type checker on expressions
shared int a = 3;
int* b = 2;

function void main() {
    int* i = 7;
    int  j = 5;
    int  k = i + j;
    print (a + b) * k;
}
