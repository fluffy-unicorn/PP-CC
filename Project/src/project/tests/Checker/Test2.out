Sprockell 0 says 60
